CMakeLists.txt

```
@@ -12,6 +12,7 @@ if(ELASTIX_USE_OPENCL)
 endif()
 set(ELASTIX_BUILD_EXECUTABLE OFF)
 set(ELASTIX_BUILD_EXECUTABLE OFF CACHE BOOL "Generate executable or library")
+set( ELASTIX_IMAGE_2D_PIXELTYPES "unsigned char;float" CACHE STRING "Specify 2D pixel types" )
```

style consistency: no space between parentheses and their content
```suggestion
set(ELASTIX_IMAGE_2D_PIXELTYPES "unsigned char;float" CACHE STRING "Specify 2D pixel types")
```